TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/InductionJourneyPage.cs

```diff
@@ -0,0 +1,11 @@
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
+
+public enum InductionJourneyPage
+{
+    Status,
+    ExemptionReason,
+    StartDate,
+    CompletedDate,
+    ChangeReason,
+    CheckYourAnswers
```

Comment:
We've called it `CheckAnswers` elsewhere
```suggestion
    CheckAnswers
```